packages/osd-ui-shared-deps/flot_charts/API.md

```diff
@@ -575,7 +575,7 @@ In Python you can get it with something like:
 calendar.timegm(datetime_object.timetuple()) * 1000
 ```
 In Ruby you can get it using the `#to_i` method on the
-[`Time`](http://apidock.com/ruby/Time/to_i) object. If you're using the
```

Comment:
This is the only real change - everything else is just whitespace autoformatting cleanup.